CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   `RenderWaveform` nodes (see `scopes` example in the demos)
 - Multiple modulo functions for the `Eval*` nodes: `mod_e` (euclidean), `mod_f`
   (floored) and `mod_t` (truncated)
+- Render to 3D textures support
```

Comment:
This "support" feels weird here. "Support for rendering to 3D textures"? Or maybe simply "Rendering to 3D textures".

Reply:
Done.